src/client.js

```diff
@@ -457,11 +458,23 @@ Client.prototype.quit = function() {
 };
 
 Client.prototype.clientAttach = function(socketId) {
+	if (_.size(this.attachedClients) === 0) {
+		this.networks.forEach(function(network) {
+			network.irc.raw("AWAY");
```

Comment:
@xPaw, I'm seeing this and wondering: could this PR be improved once #745 is merged?